### PR TITLE
Delegation Video Guide

### PR DESCRIPTION
**Pull request by Codeknight2**
Brief Video guide a created. I would guess there is a better way to embed videos, but I am no github expert.

### CODE REVIEW

**Comment by patrick-j-govalpha** [comment]
@LongForWisdom can you figure out how to get a video embedded into gitbooks?

File: delegation/mkr-holder-HowToDelegate.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Video Guide To Delegating MKR
+
+[![MKR Delegation](https://img.youtube.com/vi/_BX5VvAvzUs/0.jpg)](https://www.youtube.com/watch?v=_BX5VvAvzUs)
```
Comment (LongForWisdom) [inline delegation/mkr-holder-HowToDelegate.md:3]: Replace with: {% embed url="<iframe width="560" height="315" src="https://www.youtube.com/embed/_BX5VvAvzUs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>" %}

File: delegation/mkr-holder-HowToDelegate.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Video Guide To Delegating MKR
```
Comment (LongForWisdom) [inline delegation/mkr-holder-HowToDelegate.md:1]: Re the file-name, it should match the convention of lower-case hypen to separate. So: mkr-holder-HowToDelegate.md should become: mkr-holder-how-to-delegate.md

Comment (LongForWisdom) [reply delegation/mkr-holder-HowToDelegate.md:1]: You also need to add this to the SUMMARY.md in order for this to show up in the navigation pane.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
See comments.

File: delegation/mkr-holder-HowToDelegate.md (hunk 1 above)
Comment (LongForWisdom) [reply delegation/mkr-holder-HowToDelegate.md:3]: Nvm this doesn't work, just use https://www.youtube.com/embed/_BX5VvAvzUs

**Pull request merged**